### PR TITLE
Replace Exoscale compute endpoint /compute by /v1

### PR DESCRIPTION
**Pull request by pierre-emmanuelJ**
Replace Exoscale compute endpoint /compute by /v1

### CODE REVIEW

**Review by falzm — APPROVED** [review]

File: exoscale/client.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 
 const defaultConfig = "cloudstack.ini"
 const defaultProfile = "cloudstack"
-const defaultComputeEndpoint = "https://api.exoscale.ch/compute"
+const defaultComputeEndpoint = "https://api.exoscale.ch/v1"
```
Comment (falzm) [inline exoscale/client.go]:
```suggestion
const defaultComputeEndpoint = "https://api.exoscale.com/v1"
```

File: exoscale/client.go [hunk 2]
```diff
@@ -10,7 +10,7 @@ import (
...
 const defaultDNSEndpoint = "https://api.exoscale.ch/dns"
```
Comment (falzm) [inline exoscale/client.go]:
```suggestion
const defaultDNSEndpoint = "https://api.exoscale.com/dns"
```

**Review by falzm — CHANGES_REQUESTED** [review]
We're grown-ups now, let's switch to `.com` for the API endpoints 😉

File: exoscale/client.go (hunk 1 above)
Comment (pierre-emmanuelJ) [reply exoscale/client.go]: Nice catch

**Review by falzm — APPROVED** [review]

**Review by illi-j — APPROVED** [review]
🎂

**Pull request merged**